src/ImpurityDensities/BouleImpurityDensities/LinExpBouleImpurityDensity.jl

```diff
@@ -1,29 +1,39 @@
 """
     struct LinExpBouleImpurityDensity{T <: SSDFloat} <: AbstractImpurityDensity{T}
 
-a + b*z + c*exp((z-L)/tau)
+a + b*z + n*exp((z-l)/m)
 Impurity density model which assumes a linear + exponential component in impurity density in z.
 
 ## Fields
 * `a:T`: impurity density values at the boule origin.
 * `b::T`: linear slope in `z` direction.
-* `c::T`: exponential coefficient.
-* `L::T`: exponential offset.
-* `tau::T`: exponential scale factor.
+* `n::T`: exponential coefficient.
+* `l::T`: exponential offset.
+* `m::T`: exponential scale factor.
 * `det_z0::T`: z coordinate of the detector origin in boule coordinates. The z-direction of the detector is opposite to the z-direction of the boule coordinates.
 """
 
 struct LinExpBouleImpurityDensity{T <: SSDFloat} <: AbstractImpurityDensity{T}
     a::T
     b::T 
-    c::T 
-    L::T
-    tau::T
+    n::T 
+    l::T
+    m::T
     det_z0::T
 end
 
+function ImpurityDensity(T::DataType, t::Val{:linear_exponential_boule}, dict::AbstractDict, input_units::NamedTuple)
+    a::T = haskey(dict, "a") ? _parse_value(T, dict["a"], input_units.length^(-3)) : T(0)
+    b::T = haskey(dict, "b") ? _parse_value(T, dict["b"], input_units.length^(-4)) : T(0)
+    n::T = haskey(dict, "n") ? _parse_value(T, dict["n"], input_units.length^(-3)) : T(0)
+    l::T = haskey(dict, "l") ? _parse_value(T, dict["l"], input_units.length) : T(0)
+    m::T = haskey(dict, "m") ? _parse_value(T, dict["m"], input_units.length) : T(0)
```

Comment:
I am not sure whether you want the default value for `m` to be `T(0)` because, you will be dividing by zero then and always get `NaN`